### ISSUE TITLE
Helping in research

### ISSUE THREAD

**Issue by svebb** [xl402/node#4]
Helping in research Hello im student from Tomsk, im have in research about psychological characteristic of students, my dataset is table student data from VK, im want try use your NODE architecture, but have not enough expirience in deep lerning, maybe you can give me small and simple example how use your package, from date to prediction.

Sincerely, <@svebb00@gmail.com>

**Comment by xl402** [comment]
Hi,

Thanks for trying out this repo. As a suggestion - packages like lightgbm, XGBoost and CatBoost tend to perform better than NODE and is simpler to use, this is especially the case for CatBoost which handles categorical features out of the box as you are a beginner.

If you do want to use the NODE architecture, here is the example from the readme:
`
model = NODE(n_layers=3,
	     n_trees=100,
	     tree_depth=5,
	     units=2,
	     link=tf.keras.activations.softmax)
`
where `units` is the number of classes you have (if it is a multiclass classification problem), set it to 1 if it is binary classification.  The `link` function determines your output activation, which should be softmax if it is a classification problem. For the remaining parameters, I would recommend reading the original paper. I would like to re-iterate that NODE is probably not something you want to start off straight away if you are new to deep learning.

Best,

Tom

**Comment by svebb** [comment]
Thanks for the detailed answer and good advice.
